dist/alpine/docker-compose.yml

```diff
@@ -1,18 +1,16 @@
 version: '2.4'
 services:
   app:
-    build:                                  # Uncomment to enable local build
+    build:
+      # Uncomment to enable local build
       context: ../..
       dockerfile: ./dist/alpine/Dockerfile
       args:
         - KOPS_VERSION
         - HELM_VERSION
-    image: vathes/k8s-admin:kops${KOPS_VERSION}-helm${HELM_VERSION}-alpine
-    command: 
+    image: registry.vathes.com/vathes/k8s-admin:kops${KOPS_VERSION}-helm${HELM_VERSION}-alpine
```

Comment:
Since this image is public we published it to Docker Hub. We technically could get away with free storage on this one. :sweat_smile: